.github/CODEOWNERS

```
@@ -1,3 +1,8 @@
+# Specification files for platform maintainers
 cibuildwheel/platforms/ios.py         @freakboy3742
 cibuildwheel/platforms/pyodide.py     @hoodmane @ryanking13 @agriyakhetarpal
 cibuildwheel/platforms/android.py     @mhsmith
+
+# Constraints files
+# cibuildwheel/resources/constraints @pypa/cibuildwheel-maintainers
```

```suggestion
cibuildwheel/resources/constraints @pypa/cibuildwheel-maintainers
```

I'm unsure if this should be [`@pypa/cibuildwheel-maintainers`](https://github.com/orgs/pypa/teams/cibuildwheel-maintainers) or [`@pypa/cibuildwheel-team`](https://github.com/orgs/pypa/teams/cibuildwheel-team)